server/images.go

```diff
@@ -702,18 +702,38 @@ func convertModel(name, path string, fn func(resp api.ProgressResponse)) (string
 	return path, nil
 }
 
+func modelNameFilepath(n model.Name) (string, error) {
+	if !n.IsFullyQualified() {
```

Comment:
A name should never not be fully qualified. If it's not, it should fill the missing parts with default values

Reply:
Added back.